WordPress/WordPress.xcodeproj/project.pbxproj

```diff
@@ -8131,6 +8113,20 @@
 			shellScript = "DERIVED_SOURCES_DIR=\"${SOURCE_ROOT}/Derived Sources\"\ncp \"${SOURCE_ROOT}/Credentials/ApiCredentials.m\" \"${DERIVED_SOURCES_DIR}/ApiCredentials.m\"\n\necho \"Checking for WordPress.com Oauth App Secret in $WPCOM_CONFIG\"\nif [ -a $WPCOM_CONFIG ]\nthen\necho \"Config found\"\nsource $WPCOM_CONFIG\nelse\necho \"No config found\"\nexit 0\nfi\n\nif [ -z $WPCOM_APP_ID ]\nthen\necho \"warning: Missing WPCOM_APP_ID\"\nexit 1\nfi\nif [ -z $WPCOM_APP_SECRET ]\nthen\necho \"warning: Missing WPCOM_APP_SECRET\"\nexit 1\nfi\n\necho \"Generating credentials file in ${DERIVED_SOURCES_DIR}/ApiCredentials.m\"\nruby \"${SOURCE_ROOT}/Credentials/gencredentials.rb\" > \"${DERIVED_SOURCES_DIR}/ApiCredentials.m\"\n";
 			showEnvVarsInLog = 0;
 		};
+		E1C5456F219F10E000896227 /* Copy Gutenberg JS */ = {
+			isa = PBXShellScriptBuildPhase;
+			buildActionMask = 2147483647;
+			files = (
+			);
+			inputPaths = (
+			);
+			name = "Copy Gutenberg JS";
+			outputPaths = (
+			);
+			runOnlyForDeploymentPostprocessing = 0;
+			shellPath = /bin/sh;
+			shellScript = "# Print commands before executing them (useful for troubleshooting)\nset -x\nDEST=$CONFIGURATION_BUILD_DIR/$UNLOCALIZED_RESOURCES_FOLDER_PATH\n\nif [[ \"$CONFIGURATION\" = *Debug* && ! \"$PLATFORM_NAME\" == *simulator ]]; then\nIP=$(ipconfig getifaddr en0)\nif [ -z \"$IP\" ]; then\nIP=$(ifconfig | grep 'inet ' | grep -v ' 127.' | cut -d\\   -f2  | awk 'NR==1{print $1}')\nfi\n\necho \"$IP\" > \"$DEST/ip.txt\"\nfi\n\nBUNDLE_FILE=\"$DEST/main.jsbundle\"\ncp ${PODS_ROOT}/Gutenberg/bundle/ios/App.js ${BUNDLE_FILE}\n";
```

Comment:
As an idea, we could add the App.js file as a resource of the Pod, so we don't need a script to copy it to the app bundle.

If we do this, we would need to modify a bit the bridge delegate but it's not much.

Also, could you explain a bit what does that IP setup do?

Reply:
I tried that, but React Native seems quite strict about where the bundle has to be. At least I couldn't make it load it from there.

The IP setup is taken from this react-native script: https://github.com/facebook/react-native/blob/master/scripts/react-native-xcode.sh
That script was previously being called as a build step to create the bundle. The IP part adds a file to the bundle with the IP for the metro server, when you're running on a device (since it won't be localhost)

Reply:
`jsBundleURLForBundleRoot` will look into the mainBundle for the file. We would need to modify our bridge delegate to look into the pod's bundle for the file.

But if this works for now I think it's fine 👍
We can modify it later if it's necessary.